doc/maintaining/configuration.rst

```diff
@@ -1859,6 +1859,20 @@ example.
 Form Settings
 -------------
 
+.. ckan.dataset.create.require.resource
+
+ckan.dataset.create.require.resource
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example::
+
+ckan.dataset.create.require.resource = False
```

Comment:
You need an indentation here to avoid the Sphinx warning error

Reply:
I have made that change